Polyhedron/demo/Polyhedron/Scene_image_item.cpp

```diff
@@ -654,15 +654,33 @@ Scene_image_item::draw(Viewer_interface* viewer) const
   }
 }
 
+template <typename T> const char* whatType(T) { return "unknown"; }    // default
+template <> const char* whatType(float) { return "float"; }    // default
+template <> const char* whatType(double) { return "double"; }    // default
+template <> const char* whatType(char) { return "char"; }    // default
+template <> const char* whatType(boost::uint8_t) { return "uint8 "; }    // default
+template <> const char* whatType(boost::int16_t) { return "uint16"; }    // default
+template <> const char* whatType(boost::uint16_t type) { return "uint16_t"; }    // default
```

Comment:
The display for `uint8_t` and `uint16_t` miss the `_t` at the end.

And, to be consistent with the other integral types, you should replace `char` by `int8_t`.